### ISSUE TITLE
Layout not rendered correctly

### ISSUE THREAD

**Issue by KimTrijnh** [daattali/beautiful-jekyll#213]
Hi, 
please have a look at my page https://kimtrijnh.github.io/blog/, after forking with no changing, the page looks just plain html. 
Thanks

**Comment by daattali** [comment]
Hi Kim, in order for my theme to work out of the box, your blog needs to be hosted at <yourname>.github.io
I see you added "/blog/" to it, which means it's now a project page, not a user page. Read the README and look at the config file for settings for project pages

**Comment by KimTrijnh** [comment]
Thanks for your response. But I guess it is not the case. I don't know why forking didn't work out for me, even I used the user.github.io as in https://kimtrijnh.github.io/. I then investigated the page and found some error relating to the links to css files (shown in the img)
![image](https://user-images.githubusercontent.com/26478993/27070992-c67a4598-504d-11e7-9f40-177d21a6aa9e.png)

So, I cloned your repo and post it to another repo. Then it works https://kimtrijnh.github.io/beautiful-jekyll/. Still consfusing why forking works for others but me. 
Anw Thanks for your awesome theme
